### ISSUE TITLE
Regression test for numerical dissipation routines for z and r

### ISSUE THREAD

**Issue by mrhardman** [mabarnes/moment_kinetics#156]
The numerical dissipation routines in z and r do not use the second_derivative! function due to the need to do distributed-memory MPI reconciliation of element boundaries mid calculation. Tests of these functions are required.

@johnomotani Do such regression tests already exist?

**Comment by johnomotani** [comment]
No, I don't think we have any tests that check the distributed-MPI second derivatives. We should!
